content/applications/sales/subscriptions/closing.rst

```diff
@@ -1,105 +1,118 @@
 
-1. If you adopt such methods, you are among companies that actively and indirectly spread the
-   phobia of subscriptions, who do not think about how negatively dissatisfied customers could impact
-   their business and, who frustrate the customers in a way or another. However, at some point, it is
-   understandable that you do not want your customers to be involved in your subscription status.
-
-2. If you do not adopt such methods, you are among companies that continue to evolve in a
-   long-term subscription business model, especially in an era of ultra-fast communications between
-   dissatisfied customers, and who retain their customers by making it easier for them to leave if
-   they want to. By giving them the opportunity to close their own subscriptions, your customers do
-   not feel trapped because they subscribed to your products/services.
-
-In summary, the **Odoo Subscriptions** application gives you the
-**possibility to choose what you want to apply**. Indeed, you can decide whether to give your customers the
-option to close their subscriptions whenever they want to or to restrict this possibility. It depends
-on you and we will show you how to do that in our amazing application.
+With Odoo *Subscriptions*, companies are granted the opportunity to choose how they want
+subscriptions to close. They can give customers the control to manage their own subscriptions, or
+restrict that ability entirely.
 
 Configuration
 =============
 
-Go to :menuselection:`Subscriptions --> Configuration --> Subscription templates`. From there, you can
-create a new *Subscription template* or modify an existing one. When editing your template, underneath
-the Invoicing tab, you have the possibility to activate the option *Closable by customer*.
-Once enabled, this option gives your customers the right to close their own subscriptions.
+To properly configure the recurring plans offered, navigate to :menuselection:`Subscriptions app -->
+Configuration --> Recurring Plans`. From there, either create a new plan by clicking :guilabel:`New`
+or select an existing one to modify it.
+
+Once on the :guilabel:`Recurring Plans` form, check the box beside :guilabel:`Closable`, in the
+:guilabel:`Self-Service` section, to allow customers to close their own subscriptions using the
+customer portal.
 
-.. image:: closing/configuration-to-close-your-subscriptions.png
+.. image:: closing/recurring-plans-closable-option.png
   :align: center
-  :alt: Configuration to close your subscription with Odoo Subscriptions
+  :alt: The Closable option on a recurring plan form in Odoo Subscriptions.
 
 .. note::
-   Be sure to check out our documentation on
-   how to :doc:`Use subscription templates <plans>`
-   to fully understand the importance of this feature in a basic flow using the
-   **Odoo Subscriptions** application.
+   Check out our documentation on :doc:`subscription plans <plans>` for more information on how to
+   configure recurring plans.
```

Comment:
Very minor thing, but I was once put on blast for linking to other docs this way. Apparently, we **must** link to other doc [this way](https://www.odoo.com/documentation/16.0/contributing/documentation/rst_cheat_sheet.html#see-also)...

```suggestion
   .. seealso::
   :doc:`Configure recurring plans <plans>`
```